components/Request/Request.jsx

```diff
@@ -8,6 +8,7 @@ import { REDUCER_KEY } from '../../reducers';
 
 const Request = ({ error, matchId, loading, progress, dispatchRequest, dispatchMatchId }) => {
   function submit() {
+    document.getElementById('match_id').value = '';
```

Comment:
This is because state matchId never updated by reducer. See https://github.com/yasp-dota/ui/blob/master/reducers/request.js#L35 use `match_id` instead of `matchId` which is used by request action creator to pass action to reducer https://github.com/yasp-dota/ui/blob/master/actions/requestActions.js#L20, so state matchId won't updated to the state tree. That's why request went error. 

To overcome this problem you need to modify request reducer then modify Request.jsx to put `dispatchMatchId` after `dispatchRequest`